packages/orchestration/src/contracts/localchainAccountHolder.js

```diff
@@ -0,0 +1,154 @@
+
+const HolderI = M.interface('holder', {
+  getPublicTopics: M.call().returns(TopicsRecordShape),
+  makeDelegateInvitation: M.call(M.string(), AmountShape).returns(M.promise()),
+  makeCloseAccountInvitation: M.call().returns(M.promise()),
+  makeTransferAccountInvitation: M.call().returns(M.promise()),
+});
+
+/** @type {{ [name: string]: [description: string, valueShape: Pattern] }} */
+const PUBLIC_TOPICS = {
+  account: ['Account holder status', M.any()],
+};
+
+/**
+ * @param {import('@agoric/ertp').Baggage} baggage
+ * @param {import('@agoric/zoe/src/contractSupport/recorder.js').MakeRecorderKit} makeRecorderKit
+ * @param {ZCF} zcf
+ */
+export const prepareAccountHolder = (baggage, makeRecorderKit, zcf) => {
+  const makeAccountHolderKit = prepareExoClassKit(
+    baggage,
+    'Account Holder',
+    {
+      helper: UnguardedHelperI,
+      holder: HolderI,
+      invitationMakers: M.interface('invitationMakers', {
+        Delegate: HolderI.payload.methodGuards.makeDelegateInvitation,
+        CloseAccount: HolderI.payload.methodGuards.makeCloseAccountInvitation,
+        TransferAccount:
+          HolderI.payload.methodGuards.makeTransferAccountInvitation,
+      }),
+    },
+    /**
+     * @param {import('@agoric/vats/src/localchain.js').LocalChainAccount} account
+     * @param {StorageNode} storageNode
+     * @returns {State}
+     */
+    (account, storageNode) => {
+      // must be the fully synchronous maker because the kit is held in durable state
+      const topicKit = makeRecorderKit(storageNode, PUBLIC_TOPICS.account[1]);
+
+      return { account, topicKit };
+    },
+    {
+      helper: {
+        /** @throws if this holder no longer owns the account */
+        owned() {
+          const { account } = this.state;
+          if (!account) {
+            throw Fail`Using account holder after transfer`;
+          }
+          return account;
+        },
+        getUpdater() {
+          return this.state.topicKit.recorder;
+        },
+      },
+      invitationMakers: {
+        Delegate(validatorAddress, amount) {
+          return this.facets.holder.makeDelegateInvitation(
+            validatorAddress,
+            amount,
+          );
+        },
+        CloseAccount() {
+          return this.facets.holder.makeCloseAccountInvitation();
+        },
+        TransferAccount() {
+          return this.facets.holder.makeTransferAccountInvitation();
+        },
+      },
+      holder: {
+        getPublicTopics() {
+          const { topicKit } = this.state;
+          return harden({
+            account: {
+              description: PUBLIC_TOPICS.account[0],
+              subscriber: topicKit.subscriber,
+              storagePath: topicKit.recorder.getStoragePath(),
+            },
+          });
+        },
+        /**
+         *
+         * @param {string} validatorAddress
+         * @param {Amount<'nat'>} ertpAmount
+         */
+        async makeDelegateInvitation(validatorAddress, ertpAmount) {
```

Comment:
Likely out of scope for this PR, but thinking about how we are also accommodating contract consumers (_that do not have a smart-wallet compat requirement_).  

This accommodation might manifest self via multiple methods in the PF - one for a SW and one for a SC like `sendTo` and `makeSendInvitation` in postalSvc:
https://github.com/agoric-labs/ag-power-tools/blob/main/contract/src/postalSvc.js#L60-L61

We may instead want to treat this as just an example contract and tackle this requirement  in `vat-orchestration` .